mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/http/TelemetryClient.java

```diff
@@ -219,7 +228,7 @@ private void sendEventsWrapped(Vector<Hashtable<String, Object>> events, Callbac
     String url = getEventsEndpoint() + "/events/v2?access_token=" + getAccessToken();
 
     // Extra debug in staging mode
-    if (isStagingEnvironment()) {
+    if (isDebugLoggingEnabled()) {
```

Comment:
@zugaldia 
If we're going to go this way, we need to change

https://github.com/mapbox/mapbox-java/blob/e4842cbe98840451503a9d3153ab4124226e2211/mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/MapboxTelemetry.java#L255

with `MapboxTelemetry#setDebugLoggingEnabled(true)` and deprecate/adapt

https://github.com/mapbox/mapbox-java/blob/e4842cbe98840451503a9d3153ab4124226e2211/mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/http/TelemetryClient.java#L38

its getter and its setter

https://github.com/mapbox/mapbox-java/blob/e4842cbe98840451503a9d3153ab4124226e2211/mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/http/TelemetryClient.java#L76-L82

Another possibility is adding `|| isStagingEnvironment()` and not break anything 😉 
Right now, we're not logging when in staging mode if we don't set `debugLoggingEnabled` to `true`.

Reply:
Good call, let's do `|| isStagingEnvironment()` for now. Fixing up in a bit.